pkgs/applications/radio/cqrlog/default.nix

```diff
@@ -0,0 +1,100 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fpc
+, lazarus
+, atk
+, cairo
+, gdk-pixbuf
+, glib
+, gtk2-x11
+, libX11
+, pango
+
+, hamlib
+, mysql57
+, tqsl
+, xdg_utils
+, xplanet
+
+, autoPatchelfHook
```

Comment:
```suggestion
, xplanet
, autoPatchelfHook
```

Reply:
autoPatchelfHook isn't a propagatedBuildInputs, thus the space.

Reply:
Fixed.